src/v2/Apps/Partner/Components/PartnerContacts/PartnerContacts.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { PartnerContacts_edges } from "v2/__generated__/PartnerContacts_edges.graphql"
+import { Column, GridColumns } from "@artsy/palette"
+import { PartnerContactCardFragmentContainer as PartnerContactCard } from "./PartnerContactCard"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface ContactRouteProps {
+  edges: PartnerContacts_edges
+}
+
+export const PartnerContacts: React.FC<ContactRouteProps> = ({ edges }) => {
+  if (!edges) return null
+
+  return (
+    <GridColumns mt={40} gridRowGap={4}>
+      {edges
+        .filter(edge => !!edge && !!edge.node)
+        .map(edge => {
+          return (
+            <Column key={edge.node.id} span={12}>
+              <PartnerContactCard location={edge.node} />
+            </Column>
+          )
+        })}
```

Comment:
The previous version has some logic to reorder locations, locations from the same city are shown following each other: 
https://github.com/artsy/force/blob/0910a4645068ed363ce23864cb9384b1dad5dac6/src/desktop/apps/partner2/client/contact.coffee#L19-L23

Should we also support that?

Reply:
Great! I checked again and metaphysics is already returning an ordered list of contacts according to the CMS ordering.